File: stress/t/val_int.f
      PROGRAM VALINT
      INTEGER*8 N,S,I,P
      N=20
      S=0
      I=1
      P=1
   10 IF (I.GT.N) GOTO 20
      S=S+I
      P=P*I
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'sum=',S
      WRITE(*,'(A,I0)') 'fact=',P
      END

File: stress/t/val_int.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t n=20,s=0,i=1,p=1;
  while(i<=n){ s=s+i; p=p*i; i=i+1; }
  printf("sum="); printf("%ld",(long)s); printf("\n");
  printf("fact="); printf("%ld",(long)p); printf("\n");
  return 0;
}
